test/functionalTest/cases/2457_json_tags_with_same_name/json_tags_with_same_name.test

```diff
@@ -296,13 +296,13 @@ Date: REGEX(.*)
 07. V2: Try to create an entity E7 with attr A1 with compound value { "a": 1, "b": { "a": 1, "a": 2} } - see error
 ==================================================================================================================
 HTTP/1.1 400 Bad Request
-Content-Length: 76
+Content-Length: 75
 Content-Type: application/json
 Fiware-Correlator: REGEX([0-9a-f\-]{36})
 Date: REGEX(.*)
 
 {
-    "description": "duplicated tag-name: /a/ in path: /b/",
+    "description": "duplicated tag-name: /a/ in path: /b",
```

Comment:
The change in this .test is in a error description text, so I shouldn't be problematic from backward compatibiltiy point of view. Moreover, it can be considered a kind of bugfix, solving the unhomogenity reporting this kind of problem (step 7 and 8 where using different ways of reporting before this PR, one with final slash, the other without it).

NTC (just informative)

Reply:
Can't say I agree ...
The '/' is used instead of quotes, as quotes would need to be URI-escaped.
The slashes mark the start and end of the strings.
`/a/ in path: /b`  ... the last string is not ended. Very confusing. Seems like a bug

Reply:
So you mean chaging here (step 7) and in step 8:

/b -> /b/

?

Reply:
(Moved from general comment, @kzangeli comment)

> Yes, that would solve the problem. Also, understanding why the ending slash disappeared ... (that is what seems like a bug to me)

Reply:
Ending slash disappeared during the refactor of check(). Easy to solve.